src/evidently/ui/api/projects.py

```diff
@@ -156,7 +163,7 @@ def get_snapshot_graph_data(
     project_manager: Annotated[ProjectManager, Dependency(skip_validation=True)],
     log_event: Callable,
     user_id: UserID,
-) -> Response:
+) -> str:
```

Comment:
why

Reply:
In order to get correct types in UI:
<img width="760" alt="image" src="https://github.com/evidentlyai/evidently/assets/32310771/8edfcda6-11c5-4095-a8b3-18be0a39823f">

And this is string (not a json) because may contains NaN, +-Infinity and right now we are using special json parser